common/compositor/va/varenderer.cpp

```diff
@@ -280,7 +297,8 @@ bool VARenderer::Draw(const MediaState& state, NativeSurface* surface) {
   for (auto itr = state.colors_.begin(); itr != state.colors_.end(); itr++) {
     SetVAProcFilterColorValue(itr->first, itr->second);
   }
-  SetVAProcFilterDeinterlaceMode(state.deinterlace_);
+
+  SetVAProcFilterDeinterlaceMode(state.deinterlace_, state.layer_);
```

Comment:
Suggest this function use OverlayBuffer as the second parameter, which is clearer

Reply:
Agree this macro is not needed since HwcBuffer services both gralloc0 and gralloc1.

Reply:
Will do.